doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-expressions-syntax-dmn-based-ref.adoc

```diff
@@ -27,6 +27,11 @@ The following data types are supported by the DMN-based test scenarios in the te
 
 |===
 
+[NOTE]
+====
+An empty cell is skipped from the evaluation. To define an empty string in DMN based test scenarios, use `" "` and to define a null value, use `null`.
+====
+
 [NOTE]
 ====
```

Comment:
I'd suggest avoid using two notes back to back.

Reply:
Thank you for this suggestion. I have removed one note from the content.